wire-ios-data-model/Source/Model/Conversation/AssetCollection.swift

```diff
@@ -91,20 +91,21 @@ public class AssetCollection: NSObject, ZMCollection {
             fatal("syncMOC not accessible")
         }
         syncMOC.performGroupedBlock { [weak self] in
+            // swiftlint:disable redundant_self_in_closure
             guard let self, !self.tornDown else { return }
             guard let conversation = self.conversation,
                 let syncConversation = (try? syncMOC.existingObject(with: conversation.objectID)) as? ZMConversation else {
                 return
             }
-
-            let categorizedMessages: [ZMMessage] = AssetCollectionBatched.categorizedMessages(for: syncConversation, matchPairs: self.matchingCategories)
+            // swiftlint:enable redundant_self_in_closure
```

Comment:
These comments are not needed and even create a `superfluous_disable_command` warning.